### PR TITLE
 initial commit to add textinput field

### PR DESCRIPTION
**Pull request by rabindrachhachan**

### CODE REVIEW

**Comment by rabindrachhachan** [comment]
> Why build folder is committed?

For now, it is required to test the example app, once we published we will add build folder in gitignore file.

File: src/utils/helper.js [hunk 1]
```diff
@@ -0,0 +1,352 @@
+import _ from "lodash";
+import { isEmail, isEmpty, validateMobileNumber, isNull } from './validators';
+
+const moment = require("moment");
```
Comment (prtkkmrsngh) [inline src/utils/helper.js]: You can avoid require as we are using babel for transpiling....like this
import moment from 'moment';

File: src/fields/textInput/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+export default class TextInputField extends Component {
+    handleChange(event){
+        this.props.updateValue( this.props.attributes.name,event.target.value)
```
Comment (prtkkmrsngh) [inline src/fields/textInput/index.js]: How this keyword will be accessible without arrow function? Binding is also not there?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,286 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import TextInputField from "./fields/textInput/index.js";
+import styles from "./styles.css"
+import { autoValidate, getDefaultValue, getResetValue } from "./utils/helper";
+
+const _ = require("lodash");
```
Comment (prtkkmrsngh) [inline src/index.js]: In another file you used ES6 import for lodash then here also you can use import for lodash!

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+			fieldObj.errorMsg = '';
+			if (field && field.type) {
+				fieldObj.value = '';
+				state[field.name] = fieldObj;
+			}
+		})
+		return state;
+    }
+    
+    onValidateFields() {
```
Comment (prtkkmrsngh) [inline src/index.js]: This function also cannot access this keyword!

**Comment by prtkkmrsngh** [comment]
> > Why build folder is committed?
> 
> For now, it is required to test the example app, once we published we will add build folder in gitignore file.

ok

File: package.json [hunk 5]
```diff
@@ -1,12 +1,18 @@
 {
-  "name": "react-formo",
-  "version": "1.0.0",
+  "name": "react-jsonSchema-formo",
```
Comment (prtkkmrsngh) [inline package.json]: Why name modified?

Comment (rabindrachhachan) [reply package.json]: react-form0 already there in npm ? what will we do?

Comment (prtkkmrsngh) [reply package.json]: @sharathchandramg what you will like to keep the name...if this only then let's follow naming convention...small case!

**Pull request closed without merging**